app/experimenter/experiments/views.py

```diff
@@ -525,14 +525,14 @@ def form_valid(self, form):
         return response
 
     def form_invalid(self, form):
-        return redirect(
-            "{url}?normandy_id={value}".format(
-                url=reverse(
-                    "experiments-detail", kwargs={"slug": self.kwargs["slug"]}
-                ),
-                value=form.data["normandy_id"],
-            )
+        url = reverse(
+            "experiments-detail", kwargs={"slug": self.kwargs["slug"]}
         )
+        query_parameters = form.data.copy()
+        if query_parameters.__contains__("csrfmiddlewaretoken"):
+            query_parameters.pop("csrfmiddlewaretoken")
```

Comment:
Also is there a way to pass a fallback value to `pop` so you don't need to do the `if` check first?